Tests/IntegrationTests/PropagationTests.hpp

```diff
@@ -280,7 +280,7 @@ inline std::pair<Acts::BoundTrackParameters, double> transportToSurface(
   options_t<Actions, Aborts> options(geoCtx, magCtx);
   options.direction = Acts::Direction::Forward;
   options.pathLimit = pathLimit;
-  options.maxStepSize = 1_cm;
+  options.maxStepSize = 0.9_cm;
```

Comment:
Any reason to change this here ? Shouldn't this be part of another PR ?

Reply:
Sorry I should have mentioned this somewhere. This is a very odd fix for a problem I was facing in the integration test which compares the analytical covariance with the numerical one. For some reason the numerical derivative breaks down in the case where we step towards a surface which is 1 cm away with a single step. Using 0.9 cm it works because it will do a second step which seems to provide the necessary accuracy for the differential quotient.

This is not pretty and I would like to take a second look why a single step is not sufficient in this case.

Reply:
Do you think you could copy this information in the description of the PR ? That way in the future if we need to look at this change it will be explicit

Reply:
done :white_check_mark: